CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ ChangeLog
 
 #### Core
 
+- **New:** Added searchable table option to disable sending searchable params.
```

Comment:
```suggestion
- **New:** Added searchable table option to enable sending searchable (columns) parameters.
```
Default is `false` so the option allows enabling the feature, also added the hint that this options is a table option as we already have it as column option.